### ISSUE TITLE
Sandbox internal port not found

### ISSUE THREAD

**Issue by monkbroc** [architect/architect#1313]
**Describe the issue**
Accessing the tables client in a new application gives `Sandbox internal port not found`. The same code doesn't crash when deployed to AWS.

**Steps to reproduce**
1. Initialize a new project
```
npm init @architect bugreport
npm install @architect/functions
```

2. Define a handler at `src/http/get-index`
```
const arc = require('@architect/functions');

exports.handler = arc.http.async(handler);

async function handler(req, res) {
  await arc.tables(); // this crashes
  return {
	json: { ok: true }
  };
}
```
3. Run locally
```
npx arc sandbox
```
4. Make a request
```
curl http://localhost:3333
```

Output from `npx arc sandbox`
```
         App ⌁ bugreport
      Region ⌁ us-west-2
     Profile ⌁ @aws profile / AWS_PROFILE not configured
     Version ⌁ Architect 9.5.4
         cwd ⌁ /home/monkbroc/bugreport

✓ Sandbox Found Architect project manifest: app.arc
✓ Sandbox No testing environment variables found
✓ Sandbox Started service discovery emulator
✓ Sandbox @http server started
✓ Sandbox Project files hydrated into functions
✓ Sandbox Available @http (HTTP API mode / Lambda proxy v2.0 format) routes
    get / ................................. src/http/get-index

    http://localhost:3333

✓ Sandbox Started in 13ms
❤︎ Local environment ready!


ReferenceError: Sandbox internal port not found
    at lookup (/home/monkbroc/bugreport/node_modules/@architect/functions/src/discovery/index.js:21:32)
    at /home/monkbroc/bugreport/node_modules/@architect/functions/src/index.js:23:12
    at new Promise (<anonymous>)
    at Object.services (/home/monkbroc/bugreport/node_modules/@architect/functions/src/index.js:21:12)
    at Array.<anonymous> (/home/monkbroc/bugreport/node_modules/@architect/functions/src/tables/index.js:37:15)
    at runWaterfall (/home/monkbroc/bugreport/node_modules/run-waterfall/index.js:27:13)
    at Object.api [as tables] (/home/monkbroc/bugreport/node_modules/@architect/functions/src/tables/index.js:35:7)
    at handler (/home/monkbroc/bugreport/src/http/get-index/index.js:6:13)
    at combined (/home/monkbroc/bugreport/node_modules/@architect/functions/src/http/async/index.js:36:26)
```

**Expected behavior**
Expected no error on a new project when creating an `arc.tables()` client.

**Versions**
 - OS: Ubuntu Linux
 - Node: 14.16.1
 - Architect: 9.5.4
- package.json
```
{
  "name": "bugreport",
  "version": "0.0.0",
  "description": "A fresh new Architect project!",
  "scripts": {
    "start": "npx sandbox"
  },
  "devDependencies": {
    "@architect/architect": "^9.5.4"
  },
  "dependencies": {
    "@architect/functions": "^5.0.1"
  }
}
```

**Comment by ryanblock** [comment]
Hi @monkbroc! Sorry for this, please see the release notes in the semver major for Architect Functions v5: it requires Architect 10, which will be shipping in the next couple of days. Many moving parts to this release that made it difficult to ship them all at the same time. The best workaround would be to `npm i @architect/architect@RC` until we release, if not today then Monday.

**Comment by ryanblock** [comment]
That is correct, we hope that folks will read the changelog for semver majors but in this case I'm guessing you got this because you just installed all the latest stuff.

I'll go add a version check in Arc Functions to ensure graceful failure in the event of a version mismatch!

**Comment by ryanblock** [comment]
This shipped in `@architect/functions` v5.0.2 via https://github.com/architect/arc.codes/pull/492. Thank you again for reporting this!

**Comment by monkbroc** [comment]
Thanks for adding the version check!
